velox/common/memory/ByteStream.h

```diff
@@ -18,17 +18,19 @@
 #include "velox/common/memory/StreamArena.h"
 #include "velox/type/Type.h"
 
+#include <folly/io/IOBuf.h>
+
 namespace facebook::velox {
 
 struct ByteRange {
  public:
```

Comment:
nit: public: is not needed as member variables of a struct are public by default